app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
```

Comment:
Cool, thanks.
I tryed to build it on x86_gcc2 (primary arch) but it looks like it does not build out of the box with gcc2. So it needs a gcc2 patch. I don't know yet how many changes need to be made.
It builds fine on x86, however. I was going to build on x86_64 but since there were a couple of incorrect library version numbers, I'll add comments about them in a few minutes.